website/content/docs/platform/k8s/vso/index.mdx

```diff
@@ -121,7 +121,7 @@ Supported secrets engines: [kv-v2](/vault/docs/secrets/kv/kv-v2), [kv-v1](/vault
 
 ##### KV version 1 secret example
 
-The KV version 1 secrets engine's `kvv1` mounted path is specified under `spec.mount` of `VaultStaticSecret` custom resource. Please consult [KV Secrets Engine - Version 1 - Setup](/vault/docs/secrets/kv-v1#setup) for configuring KV secrets engine version 1. The following results in a request to `http://127.0.0.1:8200/v1/kvv1/eng/apikey/google` to retrieve the secret.
+The KV secrets engine's `kvv1` mounted path is specified under `spec.mount` of `VaultStaticSecret` custom resource. Please consult [KV Secrets Engine - Version 1 - Setup](/vault/docs/secrets/kv-v1#setup) for configuring KV secrets engine version 1. The following results in a request to `http://127.0.0.1:8200/v1/kvv1/eng/apikey/google` to retrieve the secret.
```

Comment:
I think that should be "mount path" rather than "mounted path"

Reply:
Thank you!